lib/java/com/google/android/material/chip/ChipDrawable.java

```diff
@@ -520,11 +522,37 @@ private boolean canShowCheckedIcon() {
   /** Returns the width of the chip icon plus padding, which only apply if the chip icon exists. */
   float calculateChipIconWidth() {
     if (showsChipIcon() || (showsCheckedIcon())) {
-      return iconStartPadding + chipIconSize + iconEndPadding;
+      return iconStartPadding + currentChipIconWidth() + iconEndPadding;
     }
     return 0f;
   }
 
+  /**
+   * Returns the chip's leading icon width.
+   * If chipIconUseDrawableSize is true return chipIconSize, otherwise, current icon drawable width.
```

Comment:
s/true/false

Reply:
Sorry but what means s/true/false?

Reply:
Replace "true" with "false"
The comment says "If chipIconUseDrawableSize is true return chipIconSize, otherwise, current icon drawable width."

The implementation does the opposite, if chipIconUseDrawableSize==false, then return chipIconSize, else, use the current icon drawable's width.

Reply:
Thanks! Fixed.